### PR TITLE
new recipe: Ayat_tafasir_data

### PR DESCRIPTION
**Pull request by mazbrili**
quran interpretation data for mtafsir and elforkane

### CODE REVIEW

**Comment by korli** [comment]
The recipe shouldn't be located at the root of the repository.

**Comment by mazbrili** [comment]
> The recipe shouldn't be located at the root of the repository.

sorry.. my mistake.. update it

**Comment by mazbrili** [comment]
sorry again.. wrong button click

File: app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+ * tanweer.db
+ * russian.db
+ * e3rab.db
+ * indonesian.db
+ * tafheem.db
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+           2008-2016 Tanzil.net team
+		   "
```
Comment (Begasus) [inline app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe]: Move double quote and use tab not spaces:
```diff
-           2008-2016 Tanzil.net team
-		   "
+	2008-2016 Tanzil.net team"
```

**Comment by Begasus** [comment]
@mazbrili any update here?

File: app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe (hunk 1 above)
Comment (mazbrili) [reply app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe]: ok done .. thanks @Begasus

Comment (mazbrili) [reply app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe]: > @mazbrili any update here?

.. yeah still investigate main application (elforkane) that have issue on close..
i hope i can find solution this week.. hopely

Comment (Begasus) [reply app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe]: You are still using spaces to start the line, please use a single tab instead

File: app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+	2008-2016 Tanzil.net team"
+LICENSE="Creative Commons Attribution 3.0"
+REVISION="1"
+SOURCE_URI="http://quran.ksu.edu.sa/ayat/tafasir.ayt"
+CHECKSUM_SHA256="1e96839de6ee8103ec19167f1e9668b9c44084cd0a4e7d7a33d1e05aa4e9b05d"
+SOURCE_DIR="tafasir"
+ARCHITECTURES="any"
```
Comment (korli) [inline app-text/ayat_tafasir_data/ayat_tafasir_data-0.0.1.recipe]:
```suggestion

ARCHITECTURES="any"
```

**Pull request merged**